metadata-ingestion/setup.py

```diff
@@ -430,7 +434,7 @@ def get_long_description():
     # pydantic 1.8.2 is incompatible with mypy 0.910.
     # See https://github.com/samuelcolvin/pydantic/pull/3175#issuecomment-995382910.
     "pydantic>=1.9.0",
-    "pytest>=6.2.2",
+    *pytest_common,
```

Comment:
this should be `test_api_requirements` right?

Reply:
This is what I wasn't sure about. Technically if we changed our `test_api_requirements` to not require `deepdiff`, we'd still want it in the dev requirements because we use deepdiff elsewhere (not just for the test api), so I'm not sure if I should nest the whole thing or separately declare the dependency

Reply:
So then make a separate deepdiff_dep var, which gets included both here and by test_api_requirements

That way we can remove it from `test_api_requirements` and still have it be in dev mode

(it's totally fine to have the same thing listed twice - we're using sets to dedup)